### PR TITLE
[IR] [lang] Support SAR operator: x >> y

### PR DESCRIPTION
**Pull request by Hanke98**
We support SAR operator ">>" int this PR

### CODE REVIEW

File: tests/python/test_bit_operations.py [hunk 1]
```diff
@@ -13,3 +13,13 @@ def shl(a: ti.i32, b: ti.i32) -> ti.i32:
...
+
+
+@ti.test()
+def test_bit_sar():
+    @ti.kernel
+    def sar(a: ti.i32, b: ti.i32) -> ti.i32:
+        return a >> b
+
+    for i in range(8):
+        assert sar(2**8, i) == 2**(8 - i)
```
Comment (TH3CHARLie) [inline tests/python/test_bit_operations.py:25]: Please test negative numbers here as well to ensure SAR keeps the sign bit unchanged.

**Review by TH3CHARLie — COMMENTED** [review]
Thanks for the PR! Looks nice and clean. I have only one comment about testing.

File: tests/python/test_bit_operations.py (hunk 1 above)
Comment (Hanke98) [reply tests/python/test_bit_operations.py:25]: Thanks for your advice, I will add test case for negative numbers

File: tests/python/test_bit_operations.py [hunk 2]
```diff
@@ -21,5 +21,11 @@ def test_bit_sar():
     def sar(a: ti.i32, b: ti.i32) -> ti.i32:
         return a >> b
 
+    n = 8
+    test_num = 2 ** n
+    neg_test_num = -test_num
+    for i in range(n):
+        assert sar(test_num, i) == 2**(n - i)
+    # for negative number
     for i in range(8):
```
Comment (TH3CHARLie) [inline tests/python/test_bit_operations.py:30]: Also use `n` here.

File: tests/python/test_bit_operations.py [hunk 3]
```diff
@@ -13,3 +13,19 @@ def shl(a: ti.i32, b: ti.i32) -> ti.i32:
...
+    def sar(a: ti.i32, b: ti.i32) -> ti.i32:
+        return a >> b
+
+    n = 8
+    test_num = 2 ** n
+    neg_test_num = -test_num
+    for i in range(n):
+        assert sar(test_num, i) == 2**(n - i)
+    # for negative number
+    for i in range(8):
```
Comment (TH3CHARLie) [inline tests/python/test_bit_operations.py:30]:
```suggestion
    for i in range(n):
```

File: tests/python/test_bit_operations.py (hunk 2 above)
Comment (Hanke98) [reply tests/python/test_bit_operations.py:30]: haha, sorry for that

**Comment by TH3CHARLie** [comment]
Thanks for the great work! I'll merge this once all tests are passed.

**Review by TH3CHARLie — APPROVED** [review]

**Comment by Hanke98** [comment]
> Thanks for the great work! I'll merge this once all tests are passed.

Thanks very much for your help!
